src/pb_stub.cc

```diff
@@ -464,6 +471,8 @@ Stub::Initialize(bi::managed_external_buffer::handle_t map_handle)
   py::setattr(
       python_backend_utils, "InferenceResponse",
       c_python_backend_utils.attr("InferenceResponse"));
+  py::setattr(
+      python_backend_utils, "Logger", c_python_backend_utils.attr("Logger"));
```

Comment:
I think you can remove all these lines here. They seem to be already set in `StubSetup`.

Reply:
Confirmed to still be working after removal.